katalog/fluentd/systemd.conf

```diff
@@ -33,18 +33,19 @@
 </source>
 
 # Logs from systemd-journal for interesting services.
-# <source>
-#   @type systemd
-#   @id in_systemd_bootkube
-#   matches [{ "_SYSTEMD_UNIT": "bootkube.service" }]
-#   <storage>
-#     @type local
-#     persistent true
-#     path /var/log/fluentd-journald-bootkube-cursor.json
-#   </storage>
-#   <entry>
-#     fields_strip_underscores true
-#   </entry>
-#   read_from_head true
-#   tag bootkube
-# </source>
+<source>
+  @type systemd
+  @id in_systemd_bootkube
+  matches [{ "_SYSTEMD_UNIT": "bootkube.service" }]
+  <storage>
+    @type local
+    persistent true
+    path /var/log/fluentd-journald-bootkube-cursor.json
+  </storage>
+  <entry>
+    fields_strip_underscores true
+  </entry>
+  read_from_head true
+  tag bootkube
+</source>
+
```

Comment:
We don't use bootkube, if it comes from an upstream configuration we could decide to leave it to make our life easier, but overwise we could comment it out again.

@lnovara any suggestion?

Reply:
Yes, I think we might drop it. What about adding `ssh.service`, would it make sense to have SSH logs from the node in ES too? Or is it out of scope?

Reply:
It would definitely make sense! We already have it deployed somewhere, I’ll check

Reply:
This was the configuration for systemd td-agent installation, but can be a good starting point:
```<source>
 @type tail
 @id in_tail_auth_log
 path /var/log/auth.log
 pos_file /var/log/td-agent/auth.log.pos
 tag authlog
 format /^(?<time>[^ ]* {1,2}[^ ]* [^ ]*) (?<host>[^ ]*) (?<ident>[a-zA-Z0-9_\/\.\-]*)(?:\[(?<pid>[0-9]+)\])?(?:[^\:]*\:)? *(?<message>.*)$/
</source>

<source>
  @type tail
  @id in_tail_syslog
  path /var/log/syslog
  pos_file /var/log/td-agent/syslog.pos
  tag openvpn
  <parse>
    @type syslog
  </parse>
</source>
```

Reply:
I don't want to mix things.... I think this should be addressed in another PR. Consider to create a new trello card for this please.

Reply:
I will drop the bootkube part ;)

Reply:
👍 ok for the other pr, i'll create a card for that